exporters/prometheus/exporter_test.go

```diff
@@ -939,51 +938,6 @@ func TestCollectorConcurrentSafe(t *testing.T) {
 	wg.Wait()
 }
 
-func TestIncompatibleMeterName(t *testing.T) {
-	defer func(orig otel.ErrorHandler) {
-		otel.SetErrorHandler(orig)
-	}(otel.GetErrorHandler())
-
-	errs := []error{}
-	eh := otel.ErrorHandlerFunc(func(e error) { errs = append(errs, e) })
-	otel.SetErrorHandler(eh)
-
-	// This test checks that Prometheus exporter ignores
-	// when it encounters incompatible meter name.
-
-	// Invalid label or metric name leads to error returned from
-	// createScopeInfoMetric.
-	invalidName := string([]byte{0xff, 0xfe, 0xfd})
```

Comment:
This is an invalid UTF and I do not think this test makes sense.
We would have also problems when attributes keys or values would be invalid UTF.